content/cli/overview.md

```diff
@@ -4,14 +4,18 @@ The [Nest CLI](https://github.com/nestjs/nest-cli) is a command-line interface t
 
 #### Installation
 
-**Note**: In this guide we describe using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to install packages, including the Nest CLI. Other package managers may be used at your discretion. With npm, you have several options available for managing how your OS command line resolves the location of the `nest` CLI binary file. Here, we describe installing the `nest` binary globally using the `-g` option. This provides a measure of convenience, and is the approach we assume throughout the documentation. Note that installing **any** `npm` package globally leaves the responsibility of ensuring they're running the correct version up to the user. It also means that if you have different projects, each will run the **same** version of the CLI. A reasonable alternative is to use the [npx](https://github.com/npm/npx) program (or similar features with other package managers) to ensure that you run a **managed version** of the Nest CLI. We recommend you consult the [npx documentation](https://github.com/npm/npx) and/or your DevOps support staff for more information.
+**Note**: In this guide we describe using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to install packages, including the Nest CLI. Other package managers may be used at your discretion. With npm, you have several options available for managing how your OS command line resolves the location of the `nest` CLI binary file. Here, we describe installing the `nest` binary globally using the `-g` option. This provides a measure of convenience, and is the approach we assume throughout the documentation. Note that installing **any** `npm` package globally leaves the responsibility of ensuring they're running the correct version up to the user. It also means that if you have different projects, each will run the **same** version of the CLI. A reasonable alternative is to use the [npx](https://github.com/npm/cli/blob/latest/docs/lib/content/commands/npx.md) program, built into the `npm` cli (or similar features with other package managers) to ensure that you run a **managed version** of the Nest CLI. We recommend you consult the [npx documentation](https://github.com/npm/cli/blob/latest/docs/lib/content/commands/npx.md) and/or your DevOps support staff for more information.
+
+
 
 Install the CLI globally using the `npm install -g` command (see the **Note** above for details about global installs).
 
 ```bash
 $ npm install -g @nestjs/cli
 ```
 
+> info **Hint** Alternatively, you can use this command `npx nest` without installing the cli globally. 
```

Comment:
Are you sure that npx nest will work as expected?

Reply:
My mistake, `npx nest` works if you have already installed the cli 😄, i should have written `npx @nestjs/cli@latest`, i will correct it.  I no longer have to do PR during the night 😄
```suggestion
> info **Hint** Alternatively, you can use this command `npx @nestjs/cli@latest` without installing the cli globally. 
```